src/data/checkfiles/default_checkfiles/validate-user-zone.js

```diff
@@ -1,154 +1,160 @@
-			},
-			params: {
-				op: "execute_specific_query",
-				name: specificQuery,
-				count: 0, // 0 = return all results
-				offset: 0,
-			},
-		}).catch((e) => {
-			if (!e.response || e.response.status == 400) {
-				warningAboutSpecificQuery = true;
-			}
-		});
-
-		if (resp && resp.data && resp.data.rows) {
-			resultsArr = resp.data.rows;
-			if (resultsArr.length > 0) {
-				resultsArr.map((user) => {
-					result.failed.push([user[0], user[1]]);
-				});
-			}
-		} else {
-			const generalQuery1 =
-				"SELECT USER_NAME, USER_ZONE WHERE USER_TYPE != 'rodsgroup'";
-
-			const resp1 = await axios({
-				url: `${this.httpApiLocation}/query`,
-				method: "GET",
-				headers: {
-					Authorization: `Bearer ${authToken}`,
-				},
-				params: {
-					op: "execute_genquery",
-					query: generalQuery1,
-					count: 0, // 0 = return all results
-					offset: 0,
-				},
-			});
-
-			const generalQuery2 = "SELECT ZONE_NAME";
-
-			const resp2 = await axios({
-				url: `${this.httpApiLocation}/query`,
-				method: "GET",
-				headers: {
-					Authorization: `Bearer ${authToken}`,
-				},
-				params: {
-					op: "execute_genquery",
-					query: generalQuery2,
-					count: 0, // 0 = return all results
-					offset: 0,
-				},
-			});
-
-			if (resp1 && resp1.data && resp2 && resp2.data) {
-				resp1.data.rows.map((user) => {
-					for (let i = 0; i < resp2.data.rows.length; i++) {
-						if (user[1] == resp2.data.rows[i][0]) {
-							return;
-						}
-					}
-					result.failed.push([user[0], user[1]]);
-				});
-			}
-		}
-
-		result.status = result.failed.length > 0 ? "error" : "healthy";
-
-		if (result.failed.length > 0) {
-			let pushed = (
-				<span key="push1">
-					<span>Failed on: </span>
-					{result.failed.map((failedUser, index) => (
-						<span key={`userNameCheckFailed-${index}`}>
-							{index !== 0 && ", "}
-							<Link
-								className="check_result_link"
-								key={`userNameCheckFailed-${index}`}
-								to={`/users?filter=${encodeURIComponent(
-									failedUser[0]
-								)}`}
-							>
-								{failedUser[0]}
-							</Link>{" "}
-							({failedUser[1]})
-						</span>
-					))}
-				</span>
-			);
-
-			result.message.push(pushed);
-		} else {
-			result.message.push(
-				<span key="push0">All users are in valid zones</span>
-			);
-		}
-
-		if (warningAboutSpecificQuery) {
-			result.message.push(
-				<span key="push2">
-					. Additionally, please define the specific query:{" "}
-					<Chip label={specificQuery} /> manually, or by using this{" "}
-					<a
-						rel="noreferrer"
-						target="_blank"
-						href={`/specific-query?sqlStr=${encodeURIComponent(
-							specificQuery
-						)}&alias=usersInvalidZone`}
-					>
-						dynamically generated link
-					</a>{" "}
-					to get faster results.
-				</span>
-			);
-
-			if (result.status == "healthy") {
-				result.status = "warning";
-			}
-		}
-
-		return result;
-	},
+
+const check = {
+  name: "Each user is in a valid zone.",
```

Comment:
this is the only one with a period... let's remove it for consistency.

```suggestion
  name: "Each user is in a valid zone",
```